packages/next/export/index.ts

```diff
@@ -96,19 +96,23 @@ export default async function(
   }
 
   dir = resolve(dir)
-  const nextConfig = configuration || loadConfig(PHASE_EXPORT, dir)
+  const { userConfig, config: nextConfig } =
+    configuration || loadConfig(PHASE_EXPORT, dir)
   const threads = options.threads || Math.max(cpus().length - 1, 1)
   const distDir = join(dir, nextConfig.distDir)
 
   const telemetry = options.buildExport ? null : new Telemetry({ distDir })
 
   if (telemetry) {
     telemetry.record(
-      eventVersion({
+      eventVersion(userConfig, {
         cliCommand: 'export',
         isSrcDir: null,
         hasNowJson: !!(await findUp('now.json', { cwd: dir })),
         isCustomServer: null,
+        hasTypeScript: false,
+        hasRewrites: false,
+        hasRedirects: false,
```

Comment:
This should be `null` as the status is unknown.

Reply:
I removed the fields for this case.